packages/base/src/formbuilder/index.ts

```diff
@@ -1,6 +1,6 @@
 export * from './objectform/baseform';
-export * from './objectform/geojsonsource';
-export * from './objectform/layerform';
-export * from './objectform/tilesourceform';
-export * from './objectform/vectorlayerform';
+export * from './objectform/source';
+export * from './objectform/layer';
+export * from './objectform/source';
+export * from './objectform/layer';
```

Comment:
Duplicated

```suggestion
```

Reply:
Let's turn on a linting rule that catches this too?

https://eslint.org/docs/latest/rules/no-duplicate-imports